platform/darwin/src/MGLGeoJSONSource.mm

```diff
@@ -49,9 +50,20 @@ - (instancetype)initWithIdentifier:(NSString *)identifier URL:(NSURL *)url optio
     return self;
 }
 
+- (instancetype)initWithIdentifier:(NSString *)identifier feature:(id<MGLFeature>)feature options:(NSDictionary<MGLGeoJSONSourceOption,id> *)options
```

Comment:
Any reason to require a feature instead of any ordinary shape? Looks like we’re only using this parameter for an MGLShape-typed property anyways.